docs/members.md

```diff
@@ -0,0 +1,61 @@
+# OVOS Team Members
+
+[JarbasAI](https://github.com/JarbasAl)
+
+[Daniel McKnight](https://github.com/NeonDaniel)
+
+[j1nx](https://github.com/j1nx)
+
+[forslund](https://github.com/forslund)
+
+[ChanceNCounter](https://github.com/ChanceNCounter)
+
+[5trongthany](https://github.com/5trongthany)
+
+[builderjer](https://github.com/builderjer)
+
+[goldyfruit](https://github.com/goldyfruit)
+
+[mikejgray](https://github.com/mikejgray)
+
+[emphasize](https://github.com/emphasize)
+
+[dscripka](https://github.com/dscripka)
+
+## Members hosting services
+
+### goldyfruit
+
+**use with ovos-tts-server-plugin**
+
+[Mimic1 TTS](https://tts.smartgic.io/mimic/status)
+
+[Mimic3 TTS](https://tts.smartgic.io/mimic3/status)
+
+[Piper TTS](https://tts.smartgic.io/piper/status)
+
+### builderjer
+
+**use with ovos-tts-server-plugin**
+
+[Mimic3 TTS](https://mimic3.ziggyai.online/status)
+
+[Piper TTS](https://pipertts.ziggyai.conline/status)
+
+**use with ovos-stt-server-plugin**
+
+[Faster Whisper STT](https://fasterwhisper.ziggyai.online/status)
+
+**use with ovos-tts-plugin-mimic3**
+
+[Mimic3 TTS](https://mimic3sample.ziggyai.online/api/healthcheck)
+
+### OVOS
+
+**use with ovos-tts-plugin-mimic3**
+
+[Mimic3 TTS](http://mycroft.blue-systems.com:59125/api/healthcheck)
+
+**use with ovos-stt-server-plugin**
+
+[Faster Whisper STT](https://stt.openvoiceos.org/status)
```

Comment:
Much of the purpose of Markdown is that the file is also readable within a text editor. I realize that with the advent of Pandoc and widespread HTML support, this ideal is rather unobtainable in certain situations.

However, I'd like to state that although the language may appear to enforce unnecessarily unreadable (in source) line-breaks, they're not necessary. 2 spaces after a line prevents automatic concatenation in the Markdown engines that support it, as do lists.

```suggestion
# OVOS Team Members

[JarbasAI](https://github.com/JarbasAl)  
[Daniel McKnight](https://github.com/NeonDaniel)  
[j1nx](https://github.com/j1nx)  
[forslund](https://github.com/forslund)  
[ChanceNCounter](https://github.com/ChanceNCounter)  
[5trongthany](https://github.com/5trongthany)  
[builderjer](https://github.com/builderjer)  
[goldyfruit](https://github.com/goldyfruit)  
[mikejgray](https://github.com/mikejgray)  
[emphasize](https://github.com/emphasize)  
[dscripka](https://github.com/dscripka)

## Members hosting services

### goldyfruit

**Use with `ovos-tts-server-plugin`.**

[Mimic1 TTS](https://tts.smartgic.io/mimic/status)  
[Mimic3 TTS](https://tts.smartgic.io/mimic3/status)  
[Piper TTS](https://tts.smartgic.io/piper/status)

### builderjer

**Use with `ovos-tts-server-plugin`.**

[Mimic3 TTS](https://mimic3.ziggyai.online/status)  
[Piper TTS](https://pipertts.ziggyai.conline/status)

**Use with `ovos-stt-server-plugin`.**

[Faster Whisper STT](https://fasterwhisper.ziggyai.online/status)

**Use with `ovos-tts-plugin-mimic3`.**

[Mimic3 TTS](https://mimic3sample.ziggyai.online/api/healthcheck)

### OVOS

**Use with `ovos-tts-plugin-mimic3`.**

[Mimic3 TTS](http://mycroft.blue-systems.com:59125/api/healthcheck)

**Use with `ovos-stt-server-plugin`.**

[Faster Whisper STT](https://stt.openvoiceos.org/status)
```

Reply:
note that mimic3 is NOT hosted by OVOS, it is from BlueSystems

Reply:
@JarbasAl, probably someone else should clarify that. I'm more knowledgable at grammar than I am the architecture of OVOS.